### ISSUE TITLE
Empty queries cause query_df to throw a ProgrammingError

### ISSUE THREAD

**Issue by georgipeev** [ClickHouse/clickhouse-connect#109]
Any query that results in 0 rows causes `clickhouse_connect.driver.HttpClient.query_df` to raise a `ProgrammingError`. Same query works fine if the result contains at least one row.

Python version: 3.9

clickhouse_connect version: 0.5.3 (possibly also 0.5.4)

Stack trace:
```
Traceback (most recent call last):
  File "/home/gpeev/test.py", line 9, in <module>
    df = client.query_df("select * from vendor_data where model_date = '2025-01-01'")
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/client.py", line 242, in query_df
    return pandas_result(self.query(query,
  File "/home/gpeev/venv/3.9/lib/python3.9/site-packages/clickhouse_connect/driver/query.py", line 423, in pandas_result
    raise ProgrammingError('Pandas dataframes should only be constructed from column oriented query results')
clickhouse_connect.driver.exceptions.ProgrammingError: Pandas dataframes should only be constructed from column oriented query results 
```

**Comment by genzgd** [comment]
That should actually work on 5.4?

**Comment by georgipeev** [comment]
That would be great, but 0.5.4 is broken for a different reason, #108 .

**Comment by genzgd** [comment]
Please retry against v0.5.5.  Thanks!

**Comment by georgipeev** [comment]
The issue seems to be fixed in 0.5.5. Thanks!
